### ISSUE TITLE
Inline variable type hinting causes CakePHP_Sniffs_Commenting_DocBlockAlignmentSniff

### ISSUE THREAD

**Issue by Marlinc** [cakephp/cakephp-codesniffer#141]
Using 

```php
/** @var Price $price */
            foreach ($collection->prices() as $price) {
```

Causes the ```CakePHP_Sniffs_Commenting_DocBlockAlignmentSniff``` sniff to go off.

I don't really know how to fix this.

**Comment by davidyell** [comment]
Surely if it's alignment it should just be 

```php
            /** @var Price $price */
            foreach ($collection->prices() as $price) {
```

**Comment by Marlinc** [comment]
Will it is, I just didn't copy the whitespaces in front of the comment. Its actually like this

```
            /** @var Price $price */
            foreach ($collection->prices() as $price) {
```

**Comment by ADmad** [comment]
Since the comment starts with `/**` phpcs thinks it's a API docblock. Need to somehow make it ignore such tags. IDEs suck :stuck_out_tongue:

**Comment by Marlinc** [comment]
I see, well would using another type of comment solve the issue?

**Comment by ADmad** [comment]
Pretty sure it won't complain if you use `//` or `/*`

**Comment by Marlinc** [comment]
Okay, I'll use ```/*``` instead for now.

**Comment by ADmad** [comment]
If your IDE is happy with that then this ticket should be closed. I don't think it's a bug since `/**` is meant to indicate phpcs docblock and you should use it elsewhere.

**Comment by dereuromark** [comment]
I tested it and it seems to also work with `/* ... */` in PHPStorm for example.

**Comment by Marlinc** [comment]
Yea I'm using IntelliJ IDEA and as PHPStorm is based on it appears to be fine as well. I'll close this issue as @ADmad suggested.
